### ISSUE TITLE
Can't install on mibox 

### ISSUE THREAD

**Issue by Y0SH1zzzz** [yuliskov/SmartTube#1887]
Now I have tried as much as I can think about

Installing from adb don't work
Reseting mibox doesn't work
And tested many other things but doesn't really matter when reseting doesn't work.

Why isn't this fixed when it hasn't worked for over a year?

**Comment by kinng** [comment]
Installed and worked fine on my Mibox4.  I installed the beta using the Downloader method.

**Comment by Y0SH1zzzz** [comment]
I will try again and see with that method since I didn't test that after
reseting. I'll update if I remember haha

Den mån 26 sep. 2022 07:00kinng ***@***.***> skrev:

**Comment by Y0SH1zzzz** [comment]
> Installed and worked fine on my Mibox4.  I installed the beta using the Downloader method.

Yes this did work but only after reseting my mibox which is weird
